pages/workspaces/[workspaceId]/contributor-insights/[listId]/highlights.tsx

```diff
@@ -0,0 +1,272 @@
+import React, { useEffect, useMemo, useRef, useState } from "react";
```

Comment:
I imagine the list pages under workspaces are literal copy paste from the the old list routes?

Reply:
Yes, copy/paste and updated links for navigation from within workspaces